debug/app/bespoke/smk-config.json

```diff
@@ -0,0 +1,141 @@
+    },
+    "layers": [],
+    "tools": [
+        {
+            "type": "toolbar",
+            "enabled": true
+        },
+        {
+            "type": "list-menu",
+            "enabled": true,
+            "title": "My SMK Application"
+        },
+        {
+            "type": "pan",
+            "enabled": true
+        },
+        {
+            "type": "zoom",
+            "enabled": true
+        },
+        {
+            "type": "about",
+            "enabled": false,
+            "title": "About My SMK Application",
+            "position": "list-menu",
+            "content": "<p>Welcome to SMK!</p><p>My Simple Map Kit application.</p>"
+        },
+        {
+            "type": "layers",
+            "enabled": false
+        },
+        {
+            "type": "baseMaps",
+            "enabled": false
+        },
+        {
+            "type": "identify",
+            "enabled": false,
+            "showPanel": true,
+            "showWidget": true,
+            "style": {
+                "strokeWidth": 1,
+                "strokeStyle": "1, 1",
+                "strokeColor": "#000000",
+                "strokeOpacity": 0.8,
+                "fillColor": "#000000",
+                "fillOpacity": 0.5,
+                "markerSize": [
+                    20,
+                    20
+                ],
+                "markerOffset": [
+                    10,
+                    0
+                ]
+            }
+        },
+        {
+            "order": 2,
+            "position": "toolbar",
+            "icon": "search",
+            "title": "Search for Location",
+            "showPanel": true,
+            "showLocation": true,
+            "command": {
+                "identify": true,
+                "measure": true,
+                "directions": true
+            },
+            "type": "search",
+            "enabled": false
+        },
+        {
+            "type": "coordinate",
+            "enabled": false
+        },
+        {
+            "type": "measure",
+            "enabled": false
+        },
+        {
+            "type": "markup",
+            "enabled": false
+        },
+        {
+            "type": "scale",
+            "enabled": false
+        },
+        {
+            "type": "minimap",
+            "enabled": false
+        },
+        {
+            "type": "directions",
+            "enabled": false
+        },
+        {
+            "type": "location",
+            "enabled": false
+        },
+        {
+            "type": "select",
+            "enabled": false
+        },
+        {
+            "order": 1,
+            "position": "toolbar",
+            "icon": "now_widgets",
+            "title": "Example Bespoke Tool",
+            "useComponent": true,
+            "component": {
+                "template": "<div>\n  <h3>Bespoke Tool With HTML</h3>\n  <div>\n    <p>\n      This is an example of the Bespoke tool with simple HTML\n    </p>\n  </div>\n</div>"
```

Comment:
This isn't necessary and could lead to confusion. We could actually have two Bespoke tool examples, one that demonstrates how you can add a custom template like this example and the other that uses a custom Vue component.

Reply:
In the next patch set I went with an HTML/JS-only solution (no Vue) for simplicity.

Reply:
I think we should also be demonstrating how a user can create a custom Vue component and wire that into the Bespoke tool. We can keep the simple customization that you added as this option could definitely be useful for some people, but I feel strongly that we should demonstrate the more 'powerful' option as well. Thoughts?

Reply:
I'm fine adding a Bespoke example having a Vue component, but it would need to be outside the Features area. Features expect all examples to be single-page web apps and to only display the source of the html page (via the addExample() method in /debug/feature/lib/examples.js). 

One solution would be to restore the app with a Vue-based bespoke tool to the Apps area, keep the single-page bespoke example where it is in the Features/Tool Config area, and add descriptive links to each other, so if you're looking at one, you are made aware of and can quickly navigate to the other. How does that sound?

Reply:
My vote would be to have the Bespoke examples side-side-by side in a 'Customization' category off the main debug landing page...
![image](https://user-images.githubusercontent.com/7094320/149980513-191d00d8-cc31-458f-9a97-6c93daa8abaf.png)

Upon further reflection, I'm not sure we should include the simpler example. While it does work, it is demonstrating a develop anti-pattern that I don't feel we should be promoting (the fly function gets added to the global scope). 

That's my two cents, but I'll leave it to you and Nicole to decide what goes where.